lib/reek/context/code_context.rb

```diff
@@ -20,14 +19,14 @@ class CodeContext
       extend Forwardable
       delegate each_node: :exp
       delegate %i(name type) => :exp
-      delegate %i(visibility visibility= non_public_visibility?) => :visibility_tracker
 
-      attr_reader :children, :context, :exp, :statement_counter, :visibility_tracker
+      attr_reader :children, :parent, :exp, :statement_counter
+
       private_attr_reader :refs
 
       # Initializes a new CodeContext.
       #
-      # @param context [CodeContext, nil] The parent context
+      # @param parent [CodeContext, nil] The parent context
```

Comment:
Party time! There is it! The baddest, most confusing attribute name in `Reek`s history, awesome that we change this now.
Second baddest name is "SourceCode#description" btw.